develop/application-devs/tooling/asset-transfer-api/reference.md

```diff
@@ -0,0 +1,266 @@
+
+    Learn how to install [`asset-transfer-api`](https://github.com/paritytech/asset-transfer-api){target=\_blank} into a new or existing project.
+
+    <br>
+    [:octicons-arrow-right-24: Get started](../asset-transfer-api/overview.md)
+
+-   :octicons-code-16:{ .lg .middle } __Dive in with a tutorial__
+
+    ---
+
+    Ready to start coding? Follow along with a step-by-step tutorial.
+
+    <br>
+    [:octicons-arrow-right-24: How to use the Asset Transfer API](../asset-transfer-api/overview.md/#examples)
+</div>
+<br>
+
+
+## Asset Transfer API Class
+
+Holds open an API connection to a specified chain within the `ApiPromise` to help construct transactions for assets and estimate fees.
+
+For a more in-depth explanation of the Asset Transfer API class structure, check the [source code](https://github.com/paritytech/asset-transfer-api/blob/f2aa50db83882f23492f975221dd5501c35a26d5/src/AssetTransferApi.ts#L106){target=_blank}.
+
+### Methods
+
+#### Create Transfer Transaction
+
+Generates an XCM transaction for transferring assets between chains. It simplifies the process by inferring what type of transaction is required given the inputs, ensuring that the assets are valid, and that the transaction details are correctly formatted.
+
+After obtaining the transaction, you must handle the signing and submission process separately.
+
+```ts
+--8<-- 'code/develop/application-devs/tooling/asset-transfer-api/reference/ctt-fn-signature.ts'
+```
+
+??? interface "Request parameters"
+
+    `destChainId` ++"string"++ <span class="required" markdown>++"required"++</span>
+    
+    ID of the destination chain (`'0'` for relay chain, other values for parachains).
+
+    ---
+
+    `destAddr` ++"string"++ <span class="required" markdown>++"required"++</span>
+
+    Address of the recipient account on the destination chain.
+
+    ---
+
+    `assetIds` ++"string[]"++ <span class="required" markdown>++"required"++</span>
+
+    Array of asset IDs to be transferred.
+
+    When asset IDs are provided, the API dynamically selects the appropriate pallet for the current chain to handle these specific assets. If the array is empty, the API defaults to using the `balances` pallet.
+
+    ---
+
+    `amounts` ++"string[]"++ <span class="required" markdown>++"required"++</span>
+
+    Array of amounts corresponding to each asset in `assetIds`.
+
+    ---
+
+    `opts` ++"TransferArgsOpts<T>"++
+
+    --8<-- 'code/develop/application-devs/tooling/asset-transfer-api/reference/transfer-arg-opts.md'
+
+??? interface "Response parameters"
+
+    ++"Promise<TxResult<T>"++
+
+    --8<-- 'code/develop/application-devs/tooling/asset-transfer-api/reference/tx-result.md'
+
+??? interface "Example"
+
+    ***Request***
+
+    ```ts
+    --8<-- 'code/develop/application-devs/tooling/asset-transfer-api/reference/ctt-example-request.ts'
+    ```
+
+    ***Response***
+
+    --8<-- 'code/develop/application-devs/tooling/asset-transfer-api/reference/ctt-example-response.md'
```

Comment:
Same here:

![image](https://github.com/user-attachments/assets/0445b62d-3445-49bd-b5c8-8f6a3d7e7f4f)

Reply:
And `tx ConstructedFormat<T>` is empty:

![image](https://github.com/user-attachments/assets/68cfe8e6-345c-4cda-b729-a07e542cb650)